test/code/providers/Container_ImageInventory_Class_Provider_UnitTest.cpp

```diff
@@ -93,30 +105,62 @@ class ImageInventoryTest : public CppUnit::TestFixture
 		StandardTestEnumerateInstances<mi::Container_ImageInventory_Class_Provider>(m_keyNames, context, CALL_LOCATION(errMsg));
 
 		// Get images using command line
-		CPPUNIT_ASSERT(!system("docker images -q --no-trunc > /tmp/docker_image_ids.txt"));
+		CPPUNIT_ASSERT(!system("docker images --no-trunc > /tmp/docker_image_ids.txt"));
 
 		FILE* idFile = fopen("/tmp/docker_image_ids.txt", "r");
```

Comment:
Why not create an actual temporary file (with a temporary name) in the current directory, so other users on the same machine won't stomp on this? And why not use our existing framework to auto-delete the file (even if the test aborts early)?

For auto-delete of the file, see SelfDeletingFilePath in PAL: providers/Container_DaemonEvent_Class_Provider_UnitTest.cpp.

For creation of temp files, search for CreateTempFile in pal/test. There's a bunch of occurrences there.